testkit/src/main/scala/org/bitcoins/testkit/core/gen/ScriptGenerators.scala

```diff
@@ -51,9 +51,9 @@ sealed abstract class ScriptGenerators extends BitcoinSLogger {
   def emptyScriptSignature = p2pkhScriptSignature.map(_ => EmptyScriptSignature)
 
   /**
-    * Generates a [[org.bitcoins.core.protocol.script.P2SHScriptSignature P2SHScriptSignature]]
```

Comment:
I'm only commenting here, but this applies to a bunch of places: The last word shouldn't be removed here, that screws up the link